rwgame/RWGame.cpp

```diff
@@ -412,37 +414,48 @@ int RWGame::run() {
         }
         RW_PROFILE_END();
 
-        auto now = chrono::steady_clock::now();
-        auto deltaTime = chrono::duration<float>(now - lastFrame).count();
-        lastFrame = now;
+        auto currentFrame = chrono::steady_clock::now();
+        auto frameTime =
+            chrono::duration<float>(currentFrame - lastFrame).count();
+        lastFrame = currentFrame;
 
-        if(!world->isPaused()) {
-            accumulatedTime += deltaTime;
-            world->dynamicsWorld->stepSimulation(deltaTime * timescale, kMaxPhysicsSubSteps, kPhysicsTimeStep);
+        // Clamp frameTime, so we won't freeze completely
```

Comment:
Shouldn't this be a higher constant? 100ms is not a huge amount, that's less than two frames. It should be more around the 5 frame ballpark IMHO.

Reply:
Agreed